assets/airports/kcvg.json

```diff
@@ -0,0 +1,4741 @@
+                "KCVG36L": ["^VACUB", "#330"],
+                "KCVG36C": ["^NICID", "#330"],
+                "KCVG36R": ["#004"]
+            },
+            "body": ["CVG", "GIPLE", "HOBNO", "RHOMM"],
+            "exitPoints": {
+                "BKW": ["MISSN", "JODUB", "BKW"],
+                "HVQ": ["MISSN", "JODUB", "HVQ"],
+                "JODUB": ["MISSN", "JODUB"],
+                "YRK": ["YRK"]
+            },
+            "draw": [
+                ["CVG", "GIPLE", "HOBNO", "RHOMM", "YRK*"],
+                ["RHOMM", "MISSN", "JODUB*"],
+                ["JODUB", "HVQ*"],
+                ["JODUB", "BKW*"]
+            ]
+        },
+        "ROCKT8": {
+            "icao": "ROCKT8",
+            "name": "Rocket Eight",
+            "rwy": {
+                "KCVG18L": ["_RWY18LCAMUM168", "CAMUM", "ANTEO", "CODKO", "SLUBY", "MOIER"],
+                "KCVG18C": ["FLOEE", "ANTEO", "CODKO", "SLUBY", "MOIER"],
+                "KCVG18R": ["_RWY18RFANSA203", "FANSA", "HIPAS", "JABDU", "TAPIE", "SLUBY", "MOIER"],
+                "KCVG27": ["^BONLE", "^CUNKI"],
+                "KCVG36L": ["_RWY36LCIGMA325", "^CIGMA", "HOMIX", "CERPO"],
+                "KCVG36C": ["_RWY36CCIGMA325", "^CIGMA", "HOMIX", "CERPO"],
+                "KCVG36R": ["^HAGSS", "HOMIX", "CERPO"]
+            },
+            "body": [["WADAL", "A80+"], ["FARUM", "A130+"], ["CIVLO", "A230-"], "ROCKT"],
+            "exitPoints": {
+                "AHTIY": ["CADRE", "AHTIY"],
+                "TVT": ["TVT"]
+            },
+            "draw": [
+                ["HOMIX", "CERPO", "WADAL", "FARUM", "CIVLO", "ROCKT", "TVT*"],
+                ["FANSA", "HIPAS", "JABDU", "TAPIE", "SLUBY"],
+                ["CODKO", "SLUBY", "MOIER", "WADAL"],
+                ["ROCKT", "CADRE", "AHTIY*"],
+                ["BONLE", "CUNKI", "WADAL"]
+            ]
+        },
+        "SILKS4": {
+            "icao": "SILKS4",
+            "name": "Silks Four",
+            "rwy": {
+                "KCVG18L": ["_RWY18LCAMUM168", "CAMUM", "ZEGUL"],
+                "KCVG18C": ["FLOEE", "FEBLU"],
+                "KCVG18R": ["_RWY18RFANSA203", "FANSA"],
+                "KCVG27": ["^BONLE", "CULED"],
+                "KCVG36L": ["_RWY36LHOBAL325", "HOBAL", "BOUMR", "CULED"],
+                "KCVG36C": ["_RWY36CHOBAL325", "HOBAL", "BOUMR", "CULED"],
+                "KCVG36R": ["HAGSS", "EVAME", "CEMLA", "FRTZY", "CULED"]
+            },
+            "body": ["SILKS", "TRFWA"],
+            "exitPoints": {
+                "TRFWA": []
+            },
+            "draw": [
+                ["CULED", "SILKS", "TRFWA*"],
+                ["FANSA", "SILKS"],
+                ["FEBLU", "SILKS"],
+                ["ZEGUL", "SILKS"]
+            ]
+        },
+        "WHWTR9": {
+            "icao": "WHWTR9",
+            "name": "Whitewater Nine",
+            "rwy": {
+                "KCVG9": ["#094"],
+                "KCVG18L": ["^DRUSS", "#165"],
```

Comment:
Here, are we using the fixes as seen in previous departures which are non-RNAV? The chart seems to not display those fixes for this departure, instead, using the DME readings
![image](https://user-images.githubusercontent.com/19408472/40939681-5ba201c6-683d-11e8-8eee-fc3185ef1f12.png)

Reply:
yeah, after finding the coordinates for the DME turns, I noticed they were the exact same as the fixes I had found for the RNAV SIDs, so I just used those.